jax/experimental/export/serialization_generated.py

```diff
@@ -20,20 +20,20 @@
 from flatbuffers.compat import import_numpy
 np = import_numpy()
 
-class PyTreeDefKind:
+class PyTreeDefKind(object):
```

Comment:
Do you know why FB added the `object` base here and below? It shouldn't be necessary on Python 3.X.

Reply:
My guess is that FB tries to be very conservative, or they simply have not paid attention to this detail.